pretty_gpx/test/test_bridges.py

```diff
@@ -16,24 +15,20 @@ def __core_test_bridges(path: str, gt_names: set[str]) -> None:
     """Test Bridges."""
     # GIVEN
     gpx = GpxTrack.load(path)
-    bounds = gpx.get_bounds()
 
     # WHEN
     query = OverpassQuery()
-    prepare_download_city_bridges(query, bounds)
+    prepare_download_city_bridges(query, gpx)
     query.launch_queries()
-    bridges, names = process_city_bridges(query, bounds)
-    final_bridges = get_gpx_track_bridges(bridges, names, gpx)
-
+    bridges = process_city_bridges(query, gpx)
     # THEN
-    assert_same_keys([b.name for b in final_bridges], gt_names)
+    assert_same_keys([b.name for b in bridges], gt_names)
 
 
 def test_chicago_bridges() -> None:
     """Test Chicago Bridges."""
     __core_test_bridges(os.path.join(RUNNING_DIR, "marathon_chicago.gpx"),
-                        {'BP Bridge',
```

Comment:
good to see that it now discards this pedestrian bridge above the actual track!
But i don't really understand why, since it's still close to the track

Reply:
As it checks all points of the bridges and the start and end of the bridge are far away from the track it discards it.